package.json

```diff
@@ -57,7 +57,7 @@
     "lint": "eslint . --ignore-path .gitignore",
     "transpile": "npm run clean && cup build",
     "build-test": "rm -rf dist-tests && cup build-tests",
-    "just-test": "node_modules/.bin/unitest --browser=dist-tests/browser.js --node=dist-tests/node.js",
+    "just-test": "node_modules/.bin/unitest --debug --browser=dist-tests/browser.js --node=dist-tests/node.js",
```

Comment:
What does the debug flag do, should we remove it?

Reply:
Unfortunately, [it does nothing](https://github.com/rtsao/unitest#debug-node-tests) :(.  I'll remove it.